docs/dataset-overviews/aerosol-optical-properties-macv2-sp.md

```diff
@@ -0,0 +1,49 @@
+# Aerosol optical properties/MACv2-SP
+
+## Summary
+
+**All of the text below is a draft written by a non-expert to get things moving.**
+**Please do not put any faith in its accuracy.**
+
+A first version of the aerosol optical properties will be released in the coming months.
+The methods used for producing this data are stable 
+(see [Stevens et al., 2017](https://gmd.copernicus.org/articles/10/433/2017/)).
+However, the data is entirely dependent on the emissions
+(both biomass burning and anthropogenic),
+so this dataset can only be considered stable once the short-lived emissions are stable.
+
+## Navigating the data
+
+### Recommendation for pre-industrial control
+
+Use the same approach as is used for anthropogenic emissions.
+
+### Grids and frequencies provided
+
+The aerosol optical properties provides scaling factors/parameters 
+to calculate aerosol plumes associated with aerosol emissions
+for models that do not calculate this themselves.
+The provided data is only parameters.
+As a data user, you have to calculate gridded data/input yourself using [TBD].
+
+### Variables provided
+
+Two variables can be calculated from the data:
+
+1. aerosol optical properties
+1. an adjustment to cloud-droplet population densities
+
+### Uncertainty
+
+[TBD, recommendations on parameter ranges?]
+
+## Differences from CMIP6
```

Comment:
Differences from CMIP6 or any other dataset previously available. For example, we did not have CMIP6 recommended volcanic SO2 emissions but there is a dataset that existed and was used by at least modeling group.

Reply:
Ah ok nice, good point.